packages/block-library/src/image/edit.js

```diff
@@ -587,55 +587,51 @@ export class ImageEdit extends Component {
 
 		const cleanRel = removeNewTabRel( rel );
 		const isExternal = isExternalImage( id, url );
-		const editImageIcon = ( <SVG width={ 20 } height={ 20 } viewBox="0 0 20 20"><Rect x={ 11 } y={ 3 } width={ 7 } height={ 5 } rx={ 1 } /><Rect x={ 2 } y={ 12 } width={ 7 } height={ 5 } rx={ 1 } /><Path d="M13,12h1a3,3,0,0,1-3,3v2a5,5,0,0,0,5-5h1L15,9Z" /><Path d="M4,8H3l2,3L7,8H6A3,3,0,0,1,9,5V3A5,5,0,0,0,4,8Z" /></SVG> );
 		const controls = (
 			<BlockControls>
 				<BlockAlignmentToolbar
 					value={ align }
 					onChange={ this.updateAlignment }
 				/>
+				{ url && <MediaFlow
+					mediaUpload
+					mediaURL={ url }
+					allowedTypes={ ALLOWED_MEDIA_TYPES }
+					onSelect={ this.onSelectImage }
+					onSelectURL={ this.onSelectURL }
+					noticeOperations
```

Comment:
There's probably something wrong here. This prop is passed a "true" value while it seems to be expected to be an object in the component itself?

Reply:
its the same as mediaUpload, a mistake, as it is rewritten by withNotices in the component